apps/native/app/src/screens/inbox/inbox.tsx

```diff
@@ -432,7 +472,8 @@ export const InboxScreen: NavigationFunctionComponent<{
     )
   }
 
-  // Fix for a bug in react-native-navigation where the large title is not visible on iOS with bottom tabs https://github.com/wix/react-native-navigation/issues/6717
+  // Fix for a bug in react-native-navigation where the large title is not visible on iOS with
+  // bottom tabs https://github.com/wix/react-native-navigation/issues/6717
```

Comment:
Is this still relevant? The issue is closed and since 2020. Just wondering if it could be removed?

Reply:
I added it a few weeks ago, then it was relevant. But I should check after the update you did😄

Reply:
Yes still relevant, if I take this out then I don't get large header.